sdks/java/core/pom.xml

```diff
@@ -430,6 +430,19 @@
 
     <!-- test dependencies -->
     <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-common-fn-api</artifactId>
+      <type>test-jar</type>
```

Comment:
Why both `test-jar` and `test`? I'm legitimately curious, but also previous uses of `test-jar` were wrong and replaced by use of `classifier`.